members.ts

```diff
@@ -2,56 +2,19 @@ import { Member } from "@src/types";
 
 export const members: Member[] = [
   {
-    name: "CatNose",
-    role: "CTO",
+    name: "n01e0",
+    role: "2年生",
     bio:
-      "デザインが好きなプログラマー。開発者向けの情報共有プラットフォームzenn.devを開発しています。",
-    avatarSrc: "/avatars/catnose.jpg",
+      "",
+    avatarSrc: "/avatars/n01e0.jpg",
     sources: [
-      "https://zenn.dev/catnose99/feed",
-      "https://medium.com/feed/@catnose99",
+      "https://www.feneshi.co/blog/feed",
+      "https://qiita.com/n01e0/feed",
+      "https://n01e0.hateblo.jp/feed",
     ],
     includeUrlRegex: "medium.com|zenn.dev",
```

Comment:
@n01e0 

`includeUrlRegex` は要らなそう。

> remove items which not matches includeUrlRegex

とあるので。

```suggestion
    ],
```

Reply:
🦀